src/guide/javascript.md

```diff
@@ -773,355 +439,59 @@ You can use this folder structure as a reference:
-import {
-  FilterBox,
-  OptionHash,
-  OptionPipelineEntityKind,
-  OptionPipelineStatusKind,
-  PipelineEntityKind,
-  PipelineEventFilter,
-  PipelineStatusKind,
-} from '@iroha2/data-model'
-import {
-  computed,
-  onBeforeUnmount,
-  shallowReactive,
-  shallowRef,
-} from 'vue'
-import { bytesToHex } from 'hada'
-import { client } from '../client'
-
-interface EventData {
-  hash: string
-  status: string
-}
-
-const events = shallowReactive<EventData[]>([])
-
-const currentListener = shallowRef<null | SetupEventsReturn>(null)
-
-const isListening = computed(() => !!currentListener.value)
-
-async function startListening() {
-  currentListener.value = await client.torii.listenForEvents({
-    filter: FilterBox(
-      'Pipeline',
-      PipelineEventFilter({
-        entity_kind: OptionPipelineEntityKind(
-          'Some',
-          PipelineEntityKind('Transaction'),
-        ),
-        status_kind: OptionPipelineStatusKind(
-          'Some',
-          PipelineStatusKind('Committed'),
-        ),
-        hash: OptionHash('None'),
-      }),
-    ),
-  })
-
-  currentListener.value.ee.on('event', (event) => {
-    const { hash, status } = event.as('Pipeline')
-    events.push({
-      hash: bytesToHex([...hash]),
-      status: status.match({
-        Validating: () => 'validating',
-        Committed: () => 'committed',
-        Rejected: (_reason) => 'rejected with some reason',
-      }),
-    })
-  })
-}
-
-async function stopListening() {
-  await currentListener.value?.stop()
-  currentListener.value = null
-}
-
-onBeforeUnmount(stopListening)
-</script>
-
-<template>
-  <div>
-    <h3>Listening</h3>
-
-    <p>
-      <button @click="isListening ? stopListening() : startListening()">
-        {{ isListening ? 'Stop' : 'Listen' }}
-      </button>
-    </p>
-
-    <p>Events:</p>
-
-    <ul>
-      <li v-for="{ hash, status } in events" :key="hash">
-        Transaction <code>{{ hash }}</code> status:
-        {{ status }}
-      </li>
-    </ul>
-  </div>
-</template>
-```
+<<<@/snippets/js-sdk-8-main.ts
 
-### App
-
-That's it! Finally, we just need to wrap it up with the `App.vue` component
-and the `app` entrypoint:
-
-```vue
-<!-- FILE: App.vue -->
-
-<script setup lang="ts">
-import CreateDomain from './components/CreateDomain.vue'
-import Listener from './components/Listener.vue'
-import StatusChecker from './components/StatusChecker.vue'
-</script>
-
-<template>
-  <StatusChecker />
-  <hr />
-  <CreateDomain />
-  <hr />
-  <Listener />
-</template>
-
-<style lang="scss">
-#app {
-  padding: 16px;
-  font-family: sans-serif;
-}
-</style>
-```
+:::
+
+:::info
+
+In `client.ts`, we don't import `config.json`, but do this:
 
 ```ts
-// FILE: main.ts
+import { client_config } from '../../config'
+```
 
-import { createApp } from 'vue'
-import App from './App.vue'
+It is a detail of how the source code of this application really works. For
+simplicity, read this line as the following:
```

Comment:
I added this line to avoid possible confusion like "why do they import the config from `../../config`? In the project's directory structure there is only `config.json`, and no any mention of `../../config`..."

Reply:
```suggestion
Note that you need to import the config in this way because this is how the source code of this application works. You can interpret this line as `import client_config from 'config.json'`.
```